services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -2102,11 +2095,16 @@ public void uploadFile(UploadFileInfo uploadInfo, HttpServletRequest request) th
         }
         try {
           CallInfo call = getCall(uploadInfo.getCallId());
-          LOG.info(metricMessage(uploadingUser, call, OPERATION_CALL_RECORDED, STATUS_OK, System.currentTimeMillis() - opStart, null));
+          broacastCallEvent(EVENT_CALL_RECORDED, call, uploadingUser, resource.getUploadedSize(), resource.getStatus());
+          LOG.info(metricMessage(uploadingUser, call, OPERATION_CALL_RECORDED, STATUS_OK, System.currentTimeMillis() - opStart, null, resource.getUploadedSize(), resource.getStatus()));
         } catch (InvalidCallException e) {
+          broacastCallEvent(EVENT_CALL_RECORDED, null, uploadingUser, resource.getUploadedSize(), resource.getStatus());
+          LOG.info(metricMessage(uploadingUser, null, OPERATION_CALL_RECORDED, STATUS_NOT_OK, System.currentTimeMillis() - opStart, null, resource.getUploadedSize(), resource.getStatus()));
```

Comment:
```suggestion
```

nothing needed here. This case occurs when we have a problem in metricMessage function, so it is not relevant